<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearLeccion</class>
 <widget class="QDialog" name="CrearLeccion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coline - 201513696</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Opciones</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>148</width>
         <height>92</height>
        </rect>
       </property>
       <property name="title">
        <string>Lecciones</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Seleccione Leccion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>A-Coach</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>G-Coach</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>32</y>
         <width>151</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Buscar</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBuscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>127</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Modos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnLibre">
          <property name="text">
           <string>Libre</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Participar</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSpinBox" name="cbParticiipar"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>440</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Ide</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>410</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>A-Goach</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>G-Coach</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Participar</string>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Lecciones</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableWidget" name="tblLecciones">
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>G-Coach</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Nombre</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Es A-Goach</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>Prueba1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>G-Coach</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>Prueba2</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>A-Coach</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
